authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -344,6 +391,12 @@ func getAuthInfoListForWildcard(subRevReq *authzv1.SubjectAccessReviewSpec, stor
 					finalFilteredOperations[group][subRevReq.ResourceAttributes.Resource] = verbMap
 				}
 			}
+		} else if isCustomerResourceTypeCheckAvailable && isCustomResourceAccessRequest(subRevReq, &storedOperationsMap) {
```

Comment:
Custom Resources will be absent for scenarios in L388 and L400. Aren't they also valid scenarios?